themes/theme-b2b/src/mixins.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Reusable set of styles (Tailwind classes).
+ */
+
+export const ELEVALTION_RING = 'ring-1 ring-gray-600/5';
```

Comment:
Should this be also documented?  @sebald

Reply:
becaaaaause?

Reply:
Is it a token?